manifests/role_binding.yaml

```diff
@@ -9,4 +9,16 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: vm-console-proxy
-    namespace: default
```

Comment:
nit: newline at end of file missing

Reply:
It's not missing now, but it was before this PR.